pymc3/ode/utils.py

```diff
@@ -43,12 +83,12 @@ def augment_system(ode_func, n, m):
     # Will always be 0 unless the parameter is the inital condition
     # Entry i,j is partial of y[i] wrt to p[j]
     dydp_vec = tt.vector("dydp", dtype='float64')
-    dydp_vec.tag.test_value = np.zeros(n * m, dtype='float64')
+    dydp_vec.tag.test_value = make_sens_ic(n_states, n_theta, 'float64')
 
-    dydp = dydp_vec.reshape((n, m))
+    dydp = dydp_vec.reshape((n_states, n_states + n_theta))

```

Comment:
shouldn't this fail? I thought `dydp_vec` had shape `(n_states, n_theta)`

Reply:
There are gradients w.r.t. both `y0` and `theta`, that's why it's `(n_states, n_states + n_theta)`. The `make_sens_ic` in the line above returns a `(n_states * (n_states + n_theta),)` vector.

But the docstring of it is actually wrong, so thanks!